app/assets/javascripts/topbar.js

```diff
@@ -48,19 +41,9 @@ $(document).on('mouseleave','.loc-submenu', function(){
 });
 
 function mark_active_menu() {
-  var menus = $('.menu_tab_dropdown'),
-      path = window.location.pathname + window.location.search,
-      link = $("[href='%s']".replace('%s', path));
-
-  menus.removeClass('active');
-  $("[class^='menu_tab_']").removeClass('active');
+  var link = "[href='"+window.location.pathname + window.location.search+"']";
 
-  menus.each(function(index, element) {
-    element = $(element);
-    if (element.find(link).length) {
-      element.addClass('active');
-    }
-  });
+  $('.menu_tab_dropdown ').has(link).addClass('active');
```

Comment:
Let jQuery do what it does best - finding DOM elements that match certain conditions.